stable/cert-manager/values.yaml

```diff
@@ -69,6 +69,8 @@ ingressShim: {}
   # defaultACMEChallengeType: ""
   # defaultACMEDNS01ChallengeProvider: ""
 
+priorityClassName: ""
```

Comment:
Should this be commented out by default? Do we really want priorityClassName: "" in the deployment manifest?

Reply:
Hey, `if` is evaluated as **false** if the value is empty string, so deployment will not contain `priorityClassName: ""`

Reply:
Perfect. Thanks for the confirmation.